source/RN/Kubernetes-Operator-for-PSMONGODB-RN1.5.0.rst

```diff
@@ -0,0 +1,37 @@
+.. _K8SPSMDB-1.5.0:
+
+================================================================================
+*Percona Kubernetes Operator for Percona Server for MongoDB* 1.5.0
+================================================================================
+
+:Date: September 10, 2020
+:Installation: `Installing Percona Kubernetes Operator for Percona Server for MongoDB <https://www.percona.com/doc/kubernetes-operator-for-psmongodb/index.html#installation>`_
+
+New Features
+================================================================================
+
+* :jirabug:`K8SPSMDB-233`: Automatic management of system users for MongoDB on password rotation via Secret
+* :jirabug:`K8SPSMDB-226`: Official Helm chart for the Operator
+* :jirabug:`K8SPSMDB-199`: Support multiple PSMDB minor versions by the Operator
+* :jirabug:`K8SPSMDB-198`: Fully Automate Minor Version Updates (Smart Update)
+
+Improvements
+================================================================================
+
+* :jirabug:`K8SPSMDB-192`: The ability to set the mongod cursorTimeoutMillis parameter in YAML (Thanks to user xprt64 for reporting this issue)
+* :jirabug:`K8SPSMDB-197`: Adding additional certificate SANS useful for reverse DNS lookups
+* :jirabug:`K8SPSMDB-190`: Direct API quering with "curl" instead of using "kubectl" tool in scheduled backup jobs
```

Comment:
```suggestion
* :jirabug:`K8SPSMDB-190`: Direct API quering with "curl" instead of using "kubectl" tool in scheduled backup jobs (Thanks to user phin1x for the contribution)
```